src/plugins/stake-plugins/stakePlugins.ts

```diff
@@ -1,6 +1,7 @@
+import { makeTcSaversPlugin } from './thorchainSavers/tcSavers'
```

Comment:
Can we follow the patter of naming files which export a class of object the same name as the class: e.g. `TcSaversPlugin.ts` and `UniV2StakePlugin.ts`?

Reply:
Sure. But to prevent massive conflicts, I'll add as a final commit

Reply:
I think a better convention moving forward would be to choose camelCase for names of files (regardless of file type/extension) and PascalCase for any file which exports a main class/component. This way you can tell by the name that the purpose of the file by this case rather than something redundant like it's file extension.